### DISCUSSION TITLE
Another Huion 640P

### DISCUSSION THREAD

**Discussion by Brakuu** [OpenTabletDriver/OpenTabletDriver#2691] [Q&A] [answered]
Well... OTD doesn't see my Huion 640P tablet (even though I've deleted the Huion software, the message in console about not being able to find those tablets due to it's software has disappeared, but it still can't find it. Here's my Diagnostics file.
Also followed this guide: https://github.com/OpenTabletDriver/OpenTabletDriver/discussions/2265 , but at the point of "device string reader" it doesn't find mine either. Any help? :)

[Diag.json.txt](https://github.com/OpenTabletDriver/OpenTabletDriver/files/11130365/Diag.json.txt)

**Comment by jamesbt365** [comment, marked as answer]
#2265 won't help because it isn't the same issue. You have winusb installed on the device. Please follow this entry on the wiki for further guidance on how to remove winusb from the device. https://opentabletdriver.net/Wiki/FAQ/Windows#uclogic-supported-undetected

**Reply by jamesbt365** [reply]
No, device drivers are specific to the system and thus won't mess with another system if you correct the device drivers my running tabletdrivercleanup.

**Reply by Brakuu** [reply]
Okay, that works. But now when I go into VMulti Relative Mode, none of my tablet buttons work (including tapping the pen). Are you able to help with this?

**Reply by jamesbt365** [reply]
You need to use vmulti bindings in vmulti mode, you cannot use regular mouse buttons bindings. I recommend reading the plugin wiki for setup instructions.
